spec/models/user_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+# require_relative '../app/models/user'
+
+RSpec.describe User, type: :model do
+  describe 'validations' do
+    subject { User.new(name: 'Taflon', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.') }
+
+    it 'Name should be present' do
+      subject.name = nil
+      expect(subject).to_not be_valid
+    end
+
+    it 'photo should be present' do
+      subject.photo = nil
+      expect(subject).to_not be_valid
+    end
+
+    it 'bio should be present' do
+      subject.bio = nil
+      expect(subject).to_not be_valid
+    end
+
```

Comment:
- [ ] As required for the project, kindly ensure that all your tests are passing as right now some of them are failing.
![Screenshot from 2023-07-14 17-06-28](https://github.com/BANCUNGUYE66/Blog-app/assets/83042742/50e73ff5-1462-45d0-af3c-2a64e5d0ca00)